hudi-common/src/test/java/org/apache/hudi/common/util/collection/TestRocksDBDAO.java

```diff
@@ -45,7 +45,7 @@
 /**
  * Tests RocksDB manager {@link RocksDBDAO}.
  */
-public class TestRocksDBManager {
+public class TestRocksDBDAO {
```

Comment:
I would recommend renaming `RocksDBDAO ` to be `RocksDBManager`. WDYT? Generally, `xxxDAO` exists in the web project. cc @vinothchandar

Reply:
@yanghua I'm trying to align it with the class under test, which is `RocksDBDAO`. So the pattern is to prefix it with `Test`

Reply:
@yanghua Does the comment above make sense?

Reply:
Sorry for the late reply. Yes, your explanation is reasonable. Actually, I'd like to discuss with you about the opinion of renaming `RocksDBDAO ` to `RocksDBManager `.